app/modules/main/directives/mdtTableDirective.js

```diff
@@ -220,19 +220,33 @@
                         var body = [];
                         var customCell = [];
 
+                        // Use plain JS to append content
                         _.each(clone, function (child) {
-                            var $child = angular.element(child);
 
-                            if ($child.hasClass('theadTrRow')) {
-                                headings.push($child);
-                            } else if($child.hasClass('customCell')) {
-                                customCell.push($child);
+                            if ( child.classList !== undefined ) {
```

Comment:
But Angular Material [doesn't support IE8](https://github.com/angular/material) and the support for IE11 is enough exept SVG